eng/pipelines/templates/stages/archetype-conda-release.yml

```diff
@@ -32,3 +32,26 @@ stages:
                         Get-ChildItem -Recurse $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}
                       workingDirectory: $(Pipeline.Workspace)
                       displayName: Output Visible Conda Artifacts
+      - stage: Test_${{ replace(artifact.name, '-', '_') }}_Distribution
+        displayName: 'Conda Release: ${{artifact.name}}'
+        dependsOn: Release_${{ replace(artifact.name, '-', '_') }}_To_Blob
```

Comment:
Given you look to depend on the previous stage is there a reason to break this up into a stage as opposed to just another job in the previous stage?

Reply:
Nope. I'm ripping that out.